<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCustomPlot" name="ploter" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuTendencia">
    <property name="title">
     <string>Tendencia</string>
    </property>
    <addaction name="actionIniciar"/>
    <addaction name="actionParar"/>
    <addaction name="actionReiniciar"/>
   </widget>
   <widget class="QMenu" name="menuBluetoot">
    <property name="title">
     <string>Bluetooth</string>
    </property>
    <addaction name="actionConectar"/>
    <addaction name="actionDesconectar"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuTendencia"/>
   <addaction name="menuBluetoot"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionIniciar">
   <property name="text">
    <string>Continuar</string>
   </property>
  </action>
  <action name="actionParar">
   <property name="text">
    <string>Pausar</string>
   </property>
  </action>
  <action name="actionConectar">
   <property name="text">
    <string>Conectar</string>
   </property>
  </action>
  <action name="actionDesconectar">
   <property name="text">
    <string>Desconectar</string>
   </property>
  </action>
  <action name="actionReiniciar">
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
